src/Screens/Main/Chat/Title.tsx

```diff
@@ -36,6 +38,7 @@ const Title: React.FC<Props> = ({
   openDropdown,
 }) => {
   const color = getBuddyColor(buddyId);
+  const mentor = useSelector(getMentorByUserId(buddyId));
```

Comment:
In this component, there is a remnant of old Redux.connect, where the name of the buddy is retrieved.  (see the bottom of file) And now we use also a hook-selector. It would make sense that we only use hooks or then the connect API.. 

This is OK now as it is, but would like to have a comment to one of our tickets about this. So we can refactor this components selectors to be used in same way.

Reply:
Yeah this is probably out of the scope for this PR, but I'll add a new ticket to the board!